test/extensions/filters/http/compressor/compressor_filter_speed_test.cc

```diff
@@ -111,11 +140,50 @@ CompressorFilterConfigSharedPtr makeZstdConfig(Stats::IsolatedStoreImpl& stats,
   return config;
 }
 
-static constexpr uint64_t TestDataSize = 122880;
+CompressorFilterConfigSharedPtr makeBrotliConfig(Stats::IsolatedStoreImpl& stats,
+                                                 testing::NiceMock<Runtime::MockLoader>& runtime,
+                                                 CompressionParams params) {
+
+  envoy::extensions::filters::http::compressor::v3::Compressor compressor;
+
+  const auto quality = std::get<0>(params);
+  Envoy::Compression::Compressor::CompressorFactoryPtr compressor_factory =
+      std::make_unique<MockBrotliCompressorFactory>(quality);
+  CompressorFilterConfigSharedPtr config = std::make_shared<CompressorFilterConfig>(
+      compressor, "test.", stats, runtime, std::move(compressor_factory));
+
+  return config;
+}
+
+static uint64_t TestDataSize = 122880;
 
 Buffer::OwnedImpl generateTestData() {
   Buffer::OwnedImpl data;
-  TestUtility::feedBufferWithRandomCharacters(data, TestDataSize);
+  std::string filename = std::to_string(TestDataSize) + ".data";
```

Comment:
When do we delete this file?

Reply:
This file is used to make sure that we can use same data when run multiple times. So the deletion of this file is decided by users.

Reply:
Benchmark is still a test, all files generated during tests should be deleted.

Besides, why can not get the same data with a fixed data size?

Reply:
Sorry, I didn't notice that `feedBufferWithRandomCharacters` is implemented with fixed random seed, and we don't need this file.